kyuubi-common/src/test/scala/org/apache/kyuubi/service/NoopTBinaryFrontendService.scala

```diff
@@ -17,12 +17,10 @@
 
 package org.apache.kyuubi.service
 
-class NoopThriftBinaryFrontendService(override val serverable: Serverable)
-  extends ThriftBinaryFrontendService("NoopThriftBinaryFrontendService") {
+class NoopTBinaryFrontendService(override val serverable: Serverable)
+  extends TBinaryFrontendService("NoopThriftBinaryFrontend") {
```

Comment:
`NoopThriftBinaryFrontend ` -> `NoopTBinaryFrontendService `?

Reply:
It's better to use NoopThriftBinaryFrontend, the name is mainly for logging state change, like `Servcie [NoopThriftBinaryFrontend] is stop`

Reply:
ok